config/locales/en.yml

```diff
@@ -418,20 +444,48 @@ en:
             href: "https://www.gov.uk/government/news/government-agrees-measures-with-energy-industry-to-support-vulnerable-people-through-covid-19"
```

Comment:
Remove - this is now a very old news story

Reply:
👍